doc/content/lfr/heterogeneous_single_assembly_3D/Griffin_standalone_LFR.md

```diff
@@ -0,0 +1,277 @@
+# High Fidelity Neutronics Model for Lead-cooled Fast Reactor (LFR)
+
+*Contact: Hansol Park, hansol.park@anl.gov*
+
+This VTB model provides a high fidelity neutronics model for a representative example of a lead-cooled fast reactor with an annular MOX (UPuO) fuel whose design is based on an early iteration of an LFR-prototype assembly provided by Westinghouse Electric Company, LLC [!citep](WECLFR). The original purpose of this model is to characterize the impact of various sources of uncertainties, such as theoretical and experimental uncertainties, instrumentation uncertainties, manufacturing tolerances, correlation uncertainties, and the method and simulation uncertainties, on peak cladding, fuel and coolant temperatures in the system. For this purpose, high fidelity neutronics (fine mesh heterogeneous transport - Griffin) and thermalhydraulics (computational fluid dynamics - NekRS) calculations were performed to compute hot channel factors (HCFs) [!citep](HCFReport) using a 3D heterogeneous single assembly steady-state problem. In this VTB model, only the neutronics standlone model is explained. 
```

Comment:
The first sentence is quite long. I suggest to break it into two right before `whose design is based on...`.

Reply:
Similarly the last second sentence can be broken into two before `using a 3D heterogeneous...`. The second part can be `A 3D heterogeneous single assembly steady-state model was used for this study instead of a full core model for simplicity.`

Reply:
Addressed.